flight/Modules/Actuator/actuator.c

```diff
@@ -61,6 +61,7 @@
 #define TASK_PRIORITY PIOS_THREAD_PRIO_HIGHEST
 #define FAILSAFE_TIMEOUT_MS 100
 #define MAX_MIX_ACTUATORS ACTUATORCOMMAND_CHANNEL_NUMELEM
+#define MULTIROTOR_MIXER_UPPER_BOUND 128
```

Comment:
So I notice this in conjunction with the type promotion in the uavo.. What is the intention--- do we want to allow mixer values over 100%?  Or is it just that 128 was out of range before?

I think I'd prefer 127 and keeping the old range, because it wouldn't break peoples' existing mixers.  That said, if there's a reason for over-unity mixers we should consider it.  (While it would seem superficially equivalent to increasing rate PIDs to clip sooner, I have some potential mixer changes I'm evaluating which would make the two cases different).